project/src/types/types.tsx

```diff
@@ -42,3 +42,34 @@ export type Offer = {
   title: string,
   type: string,
  }
+
+export type HostAdapted = {
+  ['avatar_url']: string,
+  id: number,
+  ['is_pro']: boolean,
+  name: string,
+}
+
+export type OfferFromServer = {
```

Comment:
```suggestion
export type OfferDTO = {
```
Есть такое понятие как Data transfer object почитай)